### DISCUSSION TITLE
Wrong values

### DISCUSSION THREAD

**Discussion by clerosch** [fl4p/batmon-ha#172] [Q&A]
Hello, i have 4 Ultimatron 12V/200Ah (parallel), installed batmon-ha with little difficulties, it works only with type dummy, but there are wrong values. On screenshot: all batteries actually have minimum 70%soc. Values jump after one minute (p.e. 191 from 1,9% to 52,3%). What could be the reason? Thanks for your answer!
![batmon1](https://github.com/fl4p/batmon-ha/assets/45568661/aa4566a1-c15d-476c-9515-bdedcefab9e8)

**Comment by fl4p** [comment]
the dummy bms generates synthetic (fake) data.
try to find the BMS with `bluetoothctl` `>scan on`

**Reply by clerosch** [reply]
Thank you. I had to install batmon-ha a news one on a raspi in the near of batteries. Now when batmon-ha started, he finds all devices, also with bluetoothctl, but the values still are wrong. The dummy bms basically generates fake data or only in this case?

**Reply by Karatzie** [reply]
Dummy is a simulated BMS with fast changing values.
Do you know what type of BMS is installed in your batteries?
Batmon only supports following type of BMS: 
JK BMS / jikong (JK02 protocol)
Daly BMS
JBD / Jiabaida/ Xiaoxiang / Overkill Solar BMS
ANT BMS
Supervolt BMS
SOK BMS

**Comment by clerosch** [comment]
Thank you for your answer. I`v thougt, dummy type is for all other bms., was happy, that there were values....  I tried all types, but got no succes. In another thread i asked now if there was found a solution for my batterie type - i`m waiting for feedback.
